packages/mantine/src/theme/CSSVariablesResolver.ts

```diff
@@ -0,0 +1,14 @@
+import {CSSVariablesResolver, ConvertCSSVariablesInput, keys} from '@mantine/core';
+
+export const resolver: CSSVariablesResolver = (theme) => {
+    const result: ConvertCSSVariablesInput = {
+        variables: {},
+        dark: {},
+        light: {},
+    };
+    keys(theme.colors).forEach((color) => {
+        result.light[`--mantine-color-${color}-light`] = theme.colors[color][color === 'gray' ? 1 : 0];
+        result.light[`--mantine-color-${color}-light-hover`] = theme.colors[color][color === 'gray' ? 2 : 0];
```

Comment:
If the color is not gray then the `light` and `light-hover` will have the same shade?